src/de.hpi.swa.graal.squeak/src/de/hpi/swa/graal/squeak/nodes/bytecodes/ReturnBytecodes.java

```diff
@@ -27,6 +35,21 @@ protected AbstractReturnNode(final CompiledCodeObject code, final int index) {
             super(code, index);
         }
 
+        protected final static AbstractSqueakObject findOnSenderChain(final ContextObject from, final ContextObject target) {
```

Comment:
This needs to be `static final` for the checkstyle test to pass.

Reply:
Thank you, done